src/tests/blueprint/t_blueprint_mesh_examples.cpp

```diff
@@ -504,16 +503,94 @@ TEST(conduit_blueprint_mesh_examples, check_gen_index_state_prop)
     EXPECT_TRUE(idx.has_path("state/time"));
 }
 
+//-----------------------------------------------------------------------------
+TEST(conduit_blueprint_mesh_examples, mesh_julia_simple_nestset)
+{
+
+    Node mesh;
+
+    Node res;
+    blueprint::mesh::examples::julia_nestsets_simple(-2.0,  2.0, // x range
+                                                     -2.0,  2.0, // y range
+                                                     0.285, 0.01, // c value
+                                                     res["simple_nestset"]);
```

Comment:
res["julia_simple_nestset"]

Reply:
@cyrush curiosity question: what does "julia" refer to?

Reply:
@rhornung67 https://en.wikipedia.org/wiki/Julia_set

Reply:
https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html#julia

Reply:
I see. This is an example. I didn't look closely enough.  Thanks!